### PR TITLE
[FEATURE] Raise version contraints

### PR DESCRIPTION
**Pull request by NeoBlack**
Raise version contraints to make extension installable with latest version of TYPO3 core 8.4

### CODE REVIEW

**Comment by fnagel** [comment]
Thanks for the PR!

You're right. Since fluid has been updated to 1.0.8 (and therefor fixed https://github.com/TYPO3Fluid/Fluid/pull/124) the extension seems to work again as expected. 

I would go ahead and release a patch release but it seems there is another minor issue in 8.4, see #13
Can you confirm?

**Comment by NeoBlack** [comment]
commented on the other issue, imho no issue as long the static template is added.

File: composer.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
   "license": "GP-2.0+",
   "require": {
     "php": ">=5.4.0,<7.1",
-    "typo3/cms-core": ">=7.3.1,<8.0"
+    "typo3/cms-core": "^7.3.1 || ^8.0"
```
Comment (fnagel) [inline composer.json]: Since the extension won't work with 8.0-8.3, this should be changed to `|| ^8.4`, right?

**Comment by fnagel** [comment]
Merged. Thanks for your help!

**Pull request closed without merging**